src/clang-c-frontend/clang_c_adjust_expr.cpp

```diff
@@ -373,6 +384,10 @@ void clang_c_adjust::adjust_float_arith(exprt &expr)
       expr.id("ieee_div");
     }
 
+    // BUG: setting rounding_mode breaks migration
+    if(t.is_vector())
+      return;
+
```

Comment:
If you don't set the rounding mode, the fp operation will always default to RNE. Why is it breaking migration?

Reply:
It breaks at the `sym_name_to_symbol` method. It seems that it is trying to initialize with `file __ESBMC_rounding_mode: {}`

Do you have any ideia? Is there any extra place to mark as an array/vector of floats?